main.tf

```
@@ -1 +1,63 @@
-# Your Terraform code goes here :)
+locals {
+  policy_name = length(var.policy_name) > 0 ? var.policy_name : "${var.name_prefix}_policy"
+}
+
+module "lacework_oci_credentials" {
+  source        = "lacework/iam-user/oci"
+  version       = ">= 0.3.0"
```

We encourage the pessimistic operator to avoid using major version bumps automatically
```suggestion
  version       = "~> 0.3"
```

---

Fixed!